content/_landing/state-and-local._en.md

```diff
@@ -1,3 +1,3 @@
 ---
-redirect_to: https://partners.login.gov/product/#state-and-local
```

Comment:
@aduth I'm not sure if this is how to handle this or if this md page should just be removed. I don't _totally_ understand how these redirects work ... especially for english. Spanish and French make more sense.

Reply:
I think this would be fine. I'm not sure the history behind this redirect. Typically we'd want to keep it if we'd had page content here before, to avoid 404s from people who had bookmarked the page. This one was added in #580 without replacing anything. Maybe it was added to just have a pretty link for https://login.gov/state-and-local

Using a root-relative link _might_ have better compatibility for Federalist and local development:

```md
redirect_to: /partners/state-and-local
```

Reply:
Also, using the root-relative link will fix one of the failing build tests, which is treating it as an external link and not able to navigate to the page, since it's not yet been published.

https://login.gov/partners/state-and-local

Reply:
🙏 thank you